xrpl4j-crypto-parent/xrpl4j-crypto-core/src/test/java/org/xrpl/xrpl4j/crypto/signing/SignatureTest.java

```diff
@@ -0,0 +1,61 @@
+package org.xrpl.xrpl4j.crypto.signing;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.junit.jupiter.api.Test;
+import org.xrpl.xrpl4j.codec.addresses.UnsignedByteArray;
+import org.xrpl.xrpl4j.model.jackson.ObjectMapperFactory;
+
+public class SignatureTest {
+
+  protected final ObjectMapper objectMapper = ObjectMapperFactory.create();
+
+  @Test
+  void serializeSignature() throws JsonProcessingException {
+
+    String hexSign = "E2ACD61C90D93433402B1F704DA38DF72876B6788C2C05B3196E14BC711AECFF14A7D6276439A1" +
+      "98D8B4880EE2DB544CF351A8CE231B3340F42F9BF1EDBF5104";
+
+    Signature signature = Signature.builder()
+      .value(UnsignedByteArray.fromHex(hexSign))
+      .build();
+    String stringJson = objectMapper.writeValueAsString(signature);
+
+    assertThat(stringJson).isEqualTo("\"" + hexSign + "\"");
+
+    Signature deserialized = objectMapper.readValue(stringJson, Signature.class);
+
+    assertThat(signature).isEqualTo(deserialized);
+
+    WrappedSignature wrappedSignature = WrappedSignature.builder()
+      .signature(signature)
+      .build();
+    String serializedWrappedSignature = objectMapper.writeValueAsString(wrappedSignature);
+
+    String serialized = "{\"signature\":\"E2ACD61C90D93433402B1F704DA38DF72876B6788C2C05B3196E14BC711AECFF14A7D627643" +
+      "9A198D8B4880EE2DB544CF351A8CE231B3340F42F9BF1EDBF5104\"}";
+    assertThat(serializedWrappedSignature).isEqualTo(serialized);
```

Comment:
Checkout this serialized object! @florent-uzio

Reply:
Ah nice, thanks @mukulljangid , once this is merged and used in the cbdc-backend we will be able to merge our UI branch.
https://github.com/xpring-eng/cbdc-admin-ui/pull/323